### PR TITLE
Create docs configuration

### PR DESCRIPTION
**Pull request by iHiD**
This is a new config file for docs that will get merged into all repos. There will be some CI rules around it:
- Unique uuids (globally)
- Unique slugs (within track)
- Valid paths (from repo apex)
- Required title
- Optional blurb

The four slugs present in this file are all required and immutable. Other slugs can be added.

This is a WIP version for Ruby.

### CODE REVIEW

**Review by SleeplessByte — APPROVED** [review]
Are there ways to link to the other doc, from a doc? Will that be URL based? 

Config content seems fine.

File: docs/config.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "docs": [
+    {
+      "uuid": "7dce5fef-d759-4292-a2b4-ba3ed65f93d7",
+      "slug": "tests",
+      "path": "docs/TESTS.md",
+      "title": "Testing on the Ruby track",
+      "blurb": "Learn how to test your Ruby exercises on Exercism Running"
```
Comment (SleeplessByte) [inline docs/config.json:8]: This sentence is weird

Comment (iHiD) [inline docs/config.json:8]:
```suggestion
      "blurb": "Learn how to test your Ruby exercises on Exercism"
```

Comment (SleeplessByte) [reply docs/config.json]: Or "how to test your solutions to Ruby exercises"? idk :P

**Comment by iHiD** [comment]
> Are there ways to link to the other doc, from a doc? Will that be URL based?

You can link to relative docs in the same folder.  (e.g. `If you don't have Ruby installed check out [the installation docs](./testing).`)

If we'd had all the docs in one repo, you could also have used relative links across docs, but the link-checker CI will fail on those now, so you'll have to use full URLs instead.

File: docs/config.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "blurb": "Learn how to test your Ruby exercises on Exercism"
+    },
+    {
+      "uuid": "8504e036-d5bb-4736-a560-52b12d40425c",
+      "slug": "installation",
+      "path": "docs/INSTALLATION.md",
+      "title": "Installing Ruby locally",
+      "blurb": "Learn how to install Ruby locally to solve Exercism's exercises on your own machine"
+    },
+        {
```
Comment (ErikSchierboom) [inline docs/config.json:17]:
```suggestion
    {
```

File: docs/config.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "uuid": "32445bf2-fc71-4352-99b4-f0834601bbd4",
+      "slug": "resources",
+      "path": "docs/RESOURCES.md",
+      "title": "Useful Ruby resources",
+      "blurb": "A collection of useful resources to help you master Ruby"
+    },
+        {
```
Comment (ErikSchierboom) [inline docs/config.json:24]:
```suggestion
    {
```

**Review by ErikSchierboom — COMMENTED** [review]
LGTM

**Review by ErikSchierboom — APPROVED** [review]
Do we allow tracks to add other docs too? E.g. the C# track has a document on its test generators.

**Comment by kytrinyx** [comment]
@ErikSchierboom Do you know what needs to be done here to get this over the finish line?

**Comment by ErikSchierboom** [comment]
@kytrinyx I do. This PR can be closed, as this file already exists: https://github.com/exercism/ruby/blob/main/docs/config.json

See https://exercism.org/docs/tracks/ruby for the end result (there wouldn't be anything there without that file)

**Pull request closed without merging**